indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -232,6 +232,12 @@
    :quantity m/string-field-mapping
    :quantity.lowercase m/string-field-mapping})
 
+(defnestedmapping granule-data-format-mapping
```

Comment:
Why is this defined as a nested map? It should just be string field.

Reply:
I was getting elasticsearch errors while it was a string field